src/shared/components/login/login.js

```diff
@@ -56,15 +56,20 @@ class Login extends Component {
   }
 
   render() {
-    const { error } = this.state;
+    let { error } = this.state;
+    if (error === 'Request failed with status code 401') {
```

Comment:
I don't think so either... maybe no one has ever opened dev tools on that page before? The warnings only display in development so I could believe it